pkg/scheduler/admission/podnodeconstraints/api/install/install.go

```diff
@@ -0,0 +1,49 @@
+package install
+
+import (
+	"github.com/golang/glog"
+
+	"k8s.io/kubernetes/pkg/api/meta"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	"github.com/openshift/origin/pkg/scheduler/admission/podnodeconstraints/api"
+	"github.com/openshift/origin/pkg/scheduler/admission/podnodeconstraints/api/v1"
+)
+
+const importPrefix = "github.com/openshift/origin/pkg/scheduler/admission/podnodeconstraints/api"
```

Comment:
It's to build a restmapper (I think) - @deads2k is this needed in this instance?